src/content/tools/pub/cmd/pub-token.md

```diff
@@ -26,13 +25,16 @@ https://some-package-repo.com/my-org/my-repo package repository requested authen
 Go to https://some-package-repo.com and log in to obtain your token. 
 ```
 
-The last line is a message the server can provide to help you obtaining a token.
-Some servers might not provide such a message.
+In this last message, the final line gives you the server that can help you
+obtain a token. Not all servers provide such this message.
+
+## Add a new credential
+
+To create a new credential, invoke `dart pub token add`.
```

Comment:
```suggestion
To add a new credential, run the command `dart pub token add`.
```

The command allows the user to _add_ a credential to the repository, but doesn't create a credential for them... It's unclear from this doc where the user gets the credential from, but I think they just make one up themselves when prompted by the `add` command